src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -115,6 +115,28 @@ public function prepare_item_for_response( $theme, $request ) {
 		$data   = array();
 		$fields = $this->get_fields_for_response( $request );
 
+		$field_mappings    = array(
+			'author'      => 'Author',
+			'author_name' => 'Author Name',
+			'author_uri'  => 'Author URI',
+			'description' => 'Description',
+			'name'        => 'Name',
+			'stylesheet'  => 'Stylesheet',
+			'template'    => 'Template',
+			'theme_uri'   => 'Theme URI',
+			'version'     => 'Version',
+		);
+		$fields_to_include = array_intersect( array_keys( $field_mappings ), $fields );
+
+		foreach ( $fields_to_include as $field ) {
+			$data[ $field ] = $theme[ $field_mappings[ $field ] ];
+		}
+
+		if ( in_array( 'screenshot', $fields, true ) ) {
+			// Using $theme->get_screenshot() with no args to get absolute URL.
+			$data['screenshot'] = $theme->get_screenshot();
```

Comment:
Let's make sure this returns an empty string, not false, when the theme does not have a screenshot.

Reply:
9b9bace07e3798930406aa48d29be99984f86999